### PR TITLE
translation: chapter_introduction/summary.md

### PR DESCRIPTION
**Pull request by krahets**
If this PR is related to coding or code translation, please fill out the checklist and paste the console outputs to the PR.

- [ ] I've tested the code and ensured the outputs are the same as the outputs of reference code.
- [ ] I've checked the code (formatting, comments, indentation, file header, etc) carefully.
- [ ] The code does not rely on a particular environment or IDE and can be executed on a standard system (Win, macOS, Ubuntu).

### CODE REVIEW

File: docs-en/chapter_introduction/summary.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-- The process of sorting poker is very similar to the insertion sort algorithm. The insertion sort algorithm is suitable for sorting small data sets.
-- The steps in currency change are essentially greedy algorithms, taking the best option that currently seems to be available at each step.
-- An algorithm is a set of instructions or operational steps to solve a specific problem in a limited amount of time, while a data structure is the way data is organized and stored in a computer.
-- Data structures and algorithms are closely intertwined. The data structure is the cornerstone of the algorithm, while the algorithm is the arena in which the data structure is utilized.
-- We can analogize data structures and algorithms to putting together building blocks, where the blocks represent the data, the shapes and connections of the blocks represent the data structure, and the steps in putting the blocks together correspond to the algorithm.
+- Algorithms are ubiquitous in daily life and are not as remote and complex as they might seem. In fact, we have already unconsciously learned many algorithms to solve various problems in life.
```
Comment (yanedie) [inline docs-en/chapter_introduction/summary.md:3]: Would it be better to substitute "inaccessible" for "remote"?

File: docs-en/chapter_introduction/summary.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+- The principle of looking up a word in a dictionary is consistent with the binary search algorithm. The binary search algorithm embodies the important algorithmic concept of divide and conquer.
+- The process of organizing playing cards is very similar to the insertion sort algorithm. The insertion sort algorithm is suitable for sorting small datasets.
+- The steps of making change in currency essentially follow the greedy algorithm, where each step involves making the best possible choice at the moment.
+- An algorithm is a set of instructions or operational steps used to solve a specific problem within a finite amount of time, while a data structure is the way data is organized and stored in a computer.
```
Comment (yanedie) [inline docs-en/chapter_introduction/summary.md:7]: Google了下，operational steps 这个词组很少见，有看到 steps of operations 的表达，更多直接就用 step。

File: docs-en/chapter_introduction/summary.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+- Data structures and algorithms are closely linked. Data structures are the foundation of algorithms, and algorithms are the stage to utilize the functions of data structures.
+- We can liken data structures and algorithms to assembling building blocks. The blocks represent data, the shape and connection method of the blocks represent data structures, and the steps of assembling the blocks correspond to algorithms.
```
Comment (yanedie) [inline docs-en/chapter_introduction/summary.md:9]: 积木首次出现在 1.2 小节中，用的是 blocks，这里用了 building blocks，可以斟酌一下要不要统一。

**Comment by krahets** [comment]
@yanedie Thanks for the review! Please check the updated documents.

**Pull request merged**